jetty-websocket/websocket-common/src/main/java/org/eclipse/jetty/websocket/common/WebSocketSessionListener.java

```diff
@@ -20,6 +20,8 @@
 
 public interface WebSocketSessionListener
 {
+    void onSessionCreated(WebSocketSession session);
```

Comment:
When adding new methods to an interface, provide a default implementation.  Actually  you can provide default implementation for all these methods now.